clang/include/clang/Basic/Attr.td

```diff
@@ -2189,6 +2189,14 @@ def BTFTypeTag : TypeAttr {
   let LangOpts = [COnly];
 }
 
+def BPFFastCall : InheritableAttr,
+                  TargetSpecificAttr<TargetBPF> {
+  let Spellings = [GCC<"bpf_fastcall">];
```

Comment:
Does GCC support this attribute with this spelling? If not, this should use the `Clang` spelling instead of the `GCC` spelling.

Reply:
GCC will have to catch-up, but the would also have to do so for `btf_{type,decl}_tag`. Changed to `Clang`.